### PR TITLE
Rishitha user management lint fix

### PR DESCRIPTION
**Pull request by RishithaMamidala**
# Description
Lint fix for the UserManagemnet Component

## Main changes explained:
All the files in src/component/UserManagement/** have been checked and all lint and prettier fixes have been made.

## How to test:
1. check into current branch
2. do npm install and ... to run this PR locally
3. log as admin user
4. Verify if the application mainly user management works as expected (Same as development branch)
5. Also check tests, npm run test

### CODE REVIEW

**Review by nikhilpittala16 — DISMISSED** [review]
I have tested the PR and everything works as expected.
Screenshot (230)
Screenshot (231)

**Comment by EvianTan** [comment]
Thanks for your efforts! Please resolve those conflicts and DM me on Slack, I'll merge it as soon as I can, or it will have conflicts again..

**Review by anirudhsk2107 — DISMISSED** [review]
Everything is working as expected 

Screen Shot 2024-10-26 at 12 40 15 AM
Screen Shot 2024-10-26 at 12 44 57 AM

**Review by Anoushka21 — APPROVED** [review]
Working as expected. No linting errors and the code is now cleaner.
image

**Comment by ShreyaLaheri** [comment]
Attached ss of what user management looks like. Not all the tests passed.
image
image

**Review by samarth9201 — APPROVED** [review]
Working as Expected.

image

**Review by Rui-Liu-github — APPROVED** [review]
Hi, I reviewed PR #2810 by pulling the latest version branch from Rishi. I checked the user management and ran test, all mentioned feature and test works as expected. 
image
image

**Comment by humera314** [comment]
Works Perfectly as Expected

image

image

**Review by SupriyaSudini — COMMENTED** [review]
I have tested the user management functionality and verified that it works the same as in the development environment.

image

User management in dev
image

I ran the test cases, and while some passed successfully, others were skipped.
Please look at the skipped test cases based on the priority
image

**Review by CodewithKoushica — CHANGES_REQUESTED** [review]
I tested this PR and there are few failed test cases. Please take a look at it.
Screenshot 2024-10-31 at 3 22 38 PM

**Comment by EvianTan** [comment]
No new issues was introduced by this PR and passed all the lint and unit tests. I'll approve and merge this PR.
image
image

**Review by EvianTan — APPROVED** [review]

File: src/components/UserManagement/ActiveCell.test.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
-import React from 'react';
-import { screen, render} from '@testing-library/react';
-import { unmountComponentAtNode } from "react-dom";
+// import React from 'react';
```
Comment (sheetalmangate) [inline src/components/UserManagement/ActiveCell.test.js:1]: Please remove commented code.

File: src/components/UserManagement/NewUserPopup.jsx [hunk 2]
```diff
@@ -25,20 +25,35 @@ const NewUserPopup = React.memo(props => {
     props.userCreated();
   };
 
+  const confirmAction = () => {
```
Comment (sheetalmangate) [inline src/components/UserManagement/NewUserPopup.jsx:28]: Remove function with no body.

File: src/components/UserManagement/NewUserPopup.jsx [hunk 3]
```diff
@@ -25,20 +25,35 @@ const NewUserPopup = React.memo(props => {
     props.userCreated();
   };
```
Comment (sheetalmangate) [inline src/components/UserManagement/NewUserPopup.jsx:27]: Remove functions that do not have body.

File: src/components/UserManagement/ResetPasswordPopup.jsx [hunk 4]
```diff
@@ -37,12 +28,12 @@ const ResetPasswordPopup = React.memo(props => {
     setError('');
   }, [props.open]);
```
Comment (sheetalmangate) [inline src/components/UserManagement/ResetPasswordPopup.jsx:30]: Please remove commented code

**Comment by sheetalmangate** [comment]
While checking functionality "Bulk Update" functionality on user management is not working. for regenerating issue
1) log in as owner 
2) Click on edit icon from header and then click on save icon.

https://github.com/user-attachments/assets/d5f2594e-4738-426b-b2c9-dd3e373eb2a7

**Pull request merged**